crates/forge/tests/it/zk/setup_script.rs

```diff
@@ -0,0 +1,48 @@
+use foundry_config::fs_permissions::PathPermission;
+use foundry_test_utils::{forgetest_async, util, util::OutputExt, TestProject};
+
+use crate::test_helpers::run_zk_script_test;
+
+forgetest_async!(setup_block_on_script_test, |prj, cmd| {
```

Comment:
Cannot get my head around the name `setup_script` vs something like `nonce` for these tests, but maybe I am missing something